deck/packages/app/src/settings.js

```diff
@@ -317,6 +317,7 @@ window.spinnakerSettings = {
     'travis',
     'webhook',
     'wercker',
+    'helm/oci',
```

Comment:
Thinking this should be helm/image vs. helm/oci.

Reply:
This is for the trigger type. Setting it same as the artifact type might create some confusion. For example all trigger types are named differently from the artifact types.